tests/std/tests/P0674R1_make_shared_for_arrays/test.cpp

```diff
@@ -1,12 +1,224 @@
 // Copyright (c) Microsoft Corporation.
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
+#include <cassert>
 #include <memory>
+#include <stdexcept>
+#include <string>
 #include <type_traits>
 #include <utility>
+#include <vector>
 
 using namespace std;
 
+#pragma warning(disable : 28251) // Inconsistent annotation for 'new': this instance has no annotations.
+
+int allocationCount = 0;
+int canCreate       = 10; // Counter to force an exception when constructing a
+                    // sufficiently large ReportAddress array
+
+struct ReportAddress;
+vector<ReportAddress*> ascendingAddressBuffer;
+vector<ReportAddress*> descendingAddressBuffer;
+
+// According to N4849, the default behaviour of new[](size) is to return

```

Comment:
We conventionally use en-us spelling. (I'll just push a change for this.)
```suggestion
// According to N4849, the default behavior of new[](size) is to return
```